src/http/http_server.cpp

```diff
@@ -38,6 +38,21 @@ DSN_DEFINE_bool("http", enable_http_server, true, "whether to enable the embedde
     }
 }
 
+/*extern*/ http_call &register_http_call(std::string full_path)
+{
+    auto call_ptr = dsn::make_unique<http_call>();
+    call_ptr->path = std::move(full_path);
+    http_call &call = *call_ptr;
+    // register this call
```

Comment:
I think the comment doesn't make sense. Maybe you can remove it.

Reply:
Sure. Any other problems?

Reply:
no